staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -27,12 +27,15 @@ import (
 	"strings"
 	"sync"
 
+	"k8s.io/apimachinery/pkg/util/httpstream"
 	"k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/pkg/api"
-	"k8s.io/client-go/pkg/kubelet/server/portforward"
-	"k8s.io/client-go/pkg/util/httpstream"
 )
 
+// TODO move to API machinery and re-unify with kubelet/server/portfoward
```

Comment:
> This does not belong to apimachinery. It's very kube specific. But I am fine with duplicating it, with a two-way reference.

If anything else wanted to do a port forward, wouldn't we encourage them to remain consistent?